<?xml version="1.0"?>
<launch>

 <!-- joy node -->
  <node name="joy_roman" pkg="joy" type="joy_node" respawn="true">
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
    <param name="autorepeat_rate" value="10.0" type="double" />
  </node>

  <!-- Parameters -->
  <rosparam command="load" file="$(find mico_pad)/launch/ps3.yaml" />
  <param name="mico_joy" value="/joy"/>
  <node name="mico_pad" pkg="mico_pad" type="mico_pad" output="screen">
  </node>
  
</launch>
